modules/ROOT/pages/software-extensibility.adoc

```diff
@@ -16,26 +16,23 @@ These extension points are guaranteed to be stable across versions of Bonita 7:
 
 === Connectors
 
-In Community, Teamwork, Efficiency, Performance and Enterprise editions
-
 Bonita comes with more than 80 xref:connectivity-overview.adoc[standard connectors] to the major information system components: major databases (Oracle, Microsoft, Postgres, etc.), SOAP Webservice, Salesforce, Email, etc.
-A new connector enables you to provide new connectivity capabilities for processes.
-To xref:connectors-overview.adoc[implement a new connector], you need to provide some XML description files and a
-Java class respecting the https://javadoc.bonitasoft.com/api/{javadocVersion}/index.html[Connector interface].
-Connectors can be implemented directly from Bonita Studio.
 
```

Comment:
To use one of them in the processes, go to the Extensions view and select the connector in the "Marketplace".
Then, configure it at task level in the "Execution" tab.